hummingbot/connector/exchange/gate_io/gate_io_utils.py

```diff
@@ -1,176 +1,3 @@
-        request_errors = True
-    return http_status, parsed_response, request_errors
-
-
-async def _sleep(delay):
-    """
-    Function added only to facilitate patching the sleep in unit tests without affecting the asyncio module
-    """
-    await asyncio.sleep(delay)
-
-
-async def api_call_with_retries(request: GateIORESTRequest,
-                                rest_assistant: RESTAssistant,
-                                throttler: AsyncThrottler,
-                                logger: logging.Logger,
-                                gate_io_auth: Optional[GateIoAuth] = None,
-                                try_count: int = 0) -> Dict[str, Any]:
-    headers = {"Content-Type": "application/json"}
-
-    async with throttler.execute_task(limit_id=request.throttler_limit_id):
-        if request.is_auth_required:
-            if gate_io_auth is None:
-                raise RuntimeError(
-                    f"Authentication required for request, but no GateIoAuth object supplied."
-                    f" Request: {request}."
-                )
-            auth_params = request.data if request.method == RESTMethod.POST else request.params
-            request.data = auth_params
-            headers: dict = gate_io_auth.get_headers(str(request.method), request.auth_url, auth_params)
-        request.headers = headers
-        response_coro = asyncio.wait_for(rest_assistant.call(request), CONSTANTS.API_CALL_TIMEOUT)
-        http_status, parsed_response, request_errors = await rest_response_with_errors(response_coro)
-
-    if request_errors or parsed_response is None:
-        if try_count < CONSTANTS.API_MAX_RETRIES:
-            try_count += 1
-            time_sleep = retry_sleep_time(try_count)
-            logger.info(
-                f"Error fetching data from {request.url}. HTTP status is {http_status}."
-                f" Retrying in {time_sleep:.0f}s."
-            )
-            await _sleep(time_sleep)
-            return await api_call_with_retries(
-                request, rest_assistant, throttler, logger, gate_io_auth, try_count
-            )
-        else:
-            raise GateIoAPIError({"label": "HTTP_ERROR", "message": parsed_response, "status": http_status})
-
-    if "message" in parsed_response:
-        raise GateIoAPIError(parsed_response)
-
-    return parsed_response
-
-
-KEYS = {
-    "gate_io_api_key":
-        ConfigVar(key="gate_io_api_key",
-                  prompt=f"Enter your {CONSTANTS.EXCHANGE_NAME} API key >>> ",
-                  required_if=using_exchange("gate_io"),
-                  is_secure=True,
-                  is_connect_key=True),
-    "gate_io_secret_key":
-        ConfigVar(key="gate_io_secret_key",
-                  prompt=f"Enter your {CONSTANTS.EXCHANGE_NAME} secret key >>> ",
-                  required_if=using_exchange("gate_io"),
-                  is_secure=True,
-                  is_connect_key=True),
-}
+from .gate_io_constants import *  # noqa: F403 F401
```

Comment:
This implementation is not clear and can confuse the community. We should keep the utils module as it was (no need to unify it with the constants)